src/views/InteractiveMap/Card.vue

```diff
@@ -189,11 +177,11 @@
                 slot="activator"
               >
                 <template v-slot:label>
-                  <div v-if="!showDiffFVAScore">
-                    Show manipulation targets
+                  <div v-if="showDiffFVAScore">
```

Comment:
Just a suggestion: you could also use a single element with a binding here, `:style="{ color: showDiffFVAScore ? 'black' : null }"`